src/Materials/Node/nodeMaterialBlockConnectionPoint.ts

```diff
@@ -146,6 +146,11 @@ export class NodeMaterialConnectionPoint {
      */
     public isOptional: boolean;
 
+    /**
+     * Gets or sets a boolean indicating that this connection point is exposed on a frame
+     */
+    public isExposedOnFrame: boolean;

```

Comment:
This is the tiny exception we did to simplify the notion of exposing a node to a frame. But all others changes in NodeMaterial that are directly linked to the editor should be challenged